sky/tools/gn

```diff
@@ -126,13 +120,10 @@ def parse_args(args):
   args = args[1:]
   parser = argparse.ArgumentParser(description='A script run` gn gen`.')
 
-  parser.add_argument('--debug', default=True, action='store_true')
-  parser.add_argument('--release', default=False, dest='debug', action='store_false')
+  parser.add_argument('--debug', default=False, action='store_true')
+
+  parser.add_argument('--runtime-mode', type=str, choices=['develop', 'profile', 'release'], default='develop')
```

Comment:
I think matching what everyone in the industry uses ("debug" and "release") is much, much more valuable than matching what Dart uses. (Search google for [debug build] and [develop build].)

I agree that there's some confusion between `flutter --debug` and `gn --debug`. I think we can mitigate this largely by referring to the latter one consistently as "engine-debug". I think that confusion will be _much_ less than the confusion that "develop" would introduce.